tests/workflows/test_pdos.py

```diff
@@ -105,35 +67,44 @@ def test_default(
     # TODO ensure this test fails if the output link from PwCalculation changes from `output_parameters` # pylint: disable=fixme
```

Comment:
Why is this added test important? The output link for the `PwCalculation` is unlikely to change, no?

Reply:
Yeh it was on the cards at some point to remove the `output_` prefix, because really its duplication having `node.outputs.output_parameters` and better to have `node.outputs.parameters`. But obviously that hasn't changed as yet

Reply:
I've removed the TODO for now. I do agree that prepending `output` to the outputs (also for the `output_structure` for example) is a bit redundant, but perhaps it's not really critical to make this (backwards-incompatible) change. I'll open an issue for this if there isn't already one where we can discuss this.